core/arch/arm/dts/stm32mp157c-ed1.dts

```diff
@@ -253,6 +253,7 @@
 			vdd_usb: ldo4 {
 				regulator-name = "vdd_usb";
 				interrupts = <IT_CURLIM_LDO4 0>;
+				regulator-always-on;
```

Comment:
The DT changes should probably go in a separate commit.
I did not quite understand the explanation about this regulator in the commit description :-/ perhaps you could reword it?

Reply:
Ok, i'll try :) sorry for my bad phrasing.